package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "loopback4-extension-grpc",
+  "version": "1.0.0-alpha.1",
+  "description": "A gRPC extencion for LoopBack Next",
+  "main": "index.js",
+  "engines": {
+    "node": ">=6"
+  },
+  "scripts": {
+    "build": "npm run build:dist && npm run build:dist6",
+    "build:dist": "./node_modules/.bin/tsc --target es2017 --outDir dist",
```

Comment:
There is no need to prefix scripts with `./node_modules/.bin`. When `npm` executes `package.json` scripts, it modifies `PATH` env variable to include the directory containing scripts from (dev) dependencies.

Reply:
ok that is weird because when I originally ran the project I had a bunch of errors for not having globally installed packages, I can try again and bring specific errors.

Reply:
We'll switch to `@loopback/build`. See https://github.com/strongloop/loopback4-extension-starter/pull/16